src/execution/sql/functions/string_functions.cpp

```diff
@@ -9,20 +9,27 @@
 
 namespace terrier::execution::sql {
 
-void StringFunctions::Concat(StringVal *result, exec::ExecutionContext *ctx, const StringVal &left,
-                             const StringVal &right) {
-  if (left.is_null_ || right.is_null_) {
+void StringFunctions::Concat(StringVal *result, exec::ExecutionContext *ctx, const int64_t num_inputs,
+                             const StringVal *inputs) {
+  if (num_inputs == 0) {
```

Comment:
Technically, this shouldn't be possible. It will be caught earlier during planning or even in code-gen. I'd change this to an assert ensuring at least one input.

Reply:
Updated